src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/custom/activity/ScheduleReductionExpirationActivityImpl.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.pu.workflow.wf.debtposition.custom.activity;
+
+import io.temporal.spring.boot.ActivityImpl;
+import it.gov.pagopa.payhub.activities.dto.debtposition.syncwfconfig.FineWfExecutionConfig;
+import it.gov.pagopa.pu.workflow.wf.debtposition.custom.fine.DebtPositionFineClient;
+import it.gov.pagopa.pu.workflow.wf.debtposition.sync.config.SynchronizeDebtPositionWfConfig;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@ActivityImpl(taskQueues = SynchronizeDebtPositionWfConfig.TASK_QUEUE_SYNCHRONIZE_DP_LOCAL_ACTIVITY)
+public class ScheduleReductionExpirationActivityImpl implements ScheduleReductionExpirationActivity {
+
+  private final DebtPositionFineClient debtPositionFineClient;
+
+  public ScheduleReductionExpirationActivityImpl(DebtPositionFineClient debtPositionFineClient) {
+    this.debtPositionFineClient = debtPositionFineClient;
+  }
+
+  @Override
+  public String expireFineReduction(Long debtPositionId, FineWfExecutionConfig wfExecutionConfig) {
+    return debtPositionFineClient.expireFineReduction(debtPositionId, wfExecutionConfig);
```

Comment:
you should not call the expireFineReduction, you should add a new method on the client to schedule the expiration! you need to add as parameter to this method the fineReductionExpirationDateTime (I think OffsetDateTime, the same type of the notificationDate on the WF)